DiscordBot/Commands/EventCommands.cs

```diff
@@ -130,13 +156,55 @@ public async Task CreateEvent(CommandContext context, string eventName, string e
         [Command("create")] 
         public async Task CreateEvent(CommandContext context, string eventName, string eventDescription, DateTime eventTime)
         {
-            await eventsSheetsService.AddEventAsync(eventName, eventDescription, eventTime);
+            var responseEmbed = new DiscordEmbedBuilder
+            {
+                Title = "Response options"
+            };
+
+
+            var i = 0;
+            foreach (var responseSet in ResponseSets)
```

Comment:
If you use a for loop instead of foreach you don't need to keep track of a separate index 🙂